### PR TITLE
String conversions

### PR DESCRIPTION
**Pull request by chfast**

### CODE REVIEW

**Comment by axic** [comment]
Will you add `hex()` here ?

**Comment by chfast** [comment]
> Will you add `hex()` here ?

Not in this PR. It requires git history rework, but looks to be ready.

File: test/unittests/test_intx.cpp [hunk 1]
```diff
@@ -167,8 +180,15 @@ TEST_F(Uint256Test, string_conversions)
     for (auto n : numbers)
     {
         auto s = to_string(n);
-        auto v = from_string<uint256>(s);
-        EXPECT_EQ(n, v);
+        try
```
Comment (chfast) [inline test/unittests/test_intx.cpp]: Revert this change.
